docs/whats-new/2020-10.md

```diff
@@ -0,0 +1,258 @@
+- [Console log formatting](../core/extensions/console-log-formatter) - Adding console log formatting article
+- [Releases and support for .NET Core and .NET 5](../core/releases-and-support) - Releases and support doc
+- [dotnet nuget verify](../core/tools/dotnet-nuget-verify) - dotnet nuget verify doc
+- [NETSDK1005 and NETSDK1047: Asset file is missing target](../core/tools/sdk-errors/netsdk1005) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1013: The TargetFramework value was not recognized](../core/tools/sdk-errors/netsdk1013) - Create netsdk1013.md
+- [NETSDK1022: Duplicate items were included.](../core/tools/sdk-errors/netsdk1022) - Create netsdk1022.md
+- [NETSDK1059: Project contains obsolete .NET CLI tool](../core/tools/sdk-errors/netsdk1059) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1071: Explicitly versioned PackageReference to a metapackage that would be included with the framework.](../core/tools/sdk-errors/netsdk1071) - Create a page for NETSDK1071
+- [NETSDK1073: The FrameworkReference was not recognized](../core/tools/sdk-errors/netsdk1073) - Create netsdk1073.md
+- [NETSDK1079: The Microsoft.AspNetCore.All package is not supported when targeting .NET Core 3.0 or higher.](../core/tools/sdk-errors/netsdk1079) - Add error message documentation for NETSDK1079
+- [Resolve NETSDK1145, targeting or apphost pack missing](../core/tools/sdk-errors/netsdk1145) - Add targeting/runtime pack missing
+
+### Updated articles
+
+- [SYSLIB0004: The constrained execution region (CER) feature is not supported](../core/compatibility/syslib0004)
+  - Update syslib0004 docs with workarounds
+  - SYSLIB reference
+- [Introduction to .NET](../core/introduction) - Update .NET intro
+- [dotnet new](../core/tools/dotnet-new) - Add missing blazorwasm options
+
+## C# language
+
+### New articles
+
+- [with expression (C# reference)](../csharp/language-reference/operators/with-expression) - C# 9 reference update: a with expression
+- [Use pattern matching to build your class behavior for better code](../csharp/tutorials/exploration/patterns-objects) - Pattern tutorial
+- [Tutorial: Explore ideas using top-level statements to build code as you learn](../csharp/tutorials/exploration/top-level-statements) - Top level statements tutorial
+- [Object-Oriented programming (C#)](../csharp/tutorials/intro-to-csharp/object-oriented-programming) - move OO tutorial to get-started
+
+### Updated articles
+
+- [Nullable reference types](../csharp/nullable-references) - Add known pitfalls to nullable reference types
+- [What's new in C# 7.0 through C# 7.3](../csharp/whats-new/csharp-7) - Consolidate c7 whats new
+
+## .NET Framework
+
+### New articles
+
+- [FindHeaderByColumn method](../framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.findheaderbycolumn) - Document private APIs in GridViewHeaderRowPresenter
```

Comment:
The first 3 were added for compliance reasons and probably shouldn't be advertised here?